features/support/capybara.rb

```diff
@@ -101,18 +101,23 @@ def is_a?(klass)
   options.add_argument('--disable-popup-blocking')
   options.add_argument('--window-size=1280,2048')
   options.add_argument('--host-resolver-rules=MAP * ~NOTFOUND , EXCLUDE *.localhost')
+  options.add_argument('--host-resolver-rules=MAP * ~NOTFOUND, EXCLUDE *localhost*')
+  options.add_argument('--disable-search-engine-choice-screen')
```

Comment:
I wonder if this was the issue with chrome 127 in CI that we faced recently. Which version of Chrome are you using locally? 

cc @mayorova

Reply:
This started to be an issue to me with ChromeDriver 128 and only when running tests with `@chrome`. AFAIK it shouldn't do anything otherwise.